src/components/Card/Card.test.tsx

```diff
@@ -1,37 +1,98 @@
 
 import { Card, CardProps } from './';
 
+import styles from './Card.module.scss';
+
 const DEFAULT_PROPS: CardProps = {
-  title: 'Title Text',
-  value: '123',
-  bottomText: 'Bottom Text'
+  title: '',
+  value: '',
+  bottomText: undefined
 };
 
+const MOCK_SKELETON_ID = 'mock-skeleton';
+
 jest.mock('../Skeleton', () => ({
-  Skeleton: () => <span>Skeleton</span>
+  Skeleton: () => <div data-testid={MOCK_SKELETON_ID} />
 }));
 
-const renderComponent = (props?: CardProps) => render(<Card {...props} />);
+describe('<Card />', () => {
+  test('should only render text in the correct order - title, value, bottom text', () => {
+    const { container } = render(<Card {...DEFAULT_PROPS} title={'1,'} value={'2,'} bottomText={'3'} />);
+    expect(container).toHaveTextContent('1,2,3');
+  });
 
-test('should work with synchronous data', () => {
-  const { getByText } = renderComponent(DEFAULT_PROPS);
-  expect(getByText('Title Text')).toBeInTheDocument();
-  expect(getByText('123')).toBeInTheDocument();
-  expect(getByText('Bottom Text')).toBeInTheDocument();
-});
+  describe('semantic tags', () => {
+    test('should render title as <label>', () => {
+      const { container } = render(<Card {...DEFAULT_PROPS} title={'mock title'} />);
+      const label = container.getElementsByTagName('label');
+      expect(label.length).toBe(1);
+      expect(label[0]).toHaveTextContent('mock title');
+    });
+  });
 
-test('should show skeleton when loading asynchronous data', () => {
-  const { getByText } = renderComponent({ ...DEFAULT_PROPS, value: { isLoading: true } });
-  expect(getByText('Skeleton')).toBeInTheDocument();
-});
+  describe('class names', () => {
+    test('should apply Title class to only the title element', () => {
+      const { container } = render(<Card {...DEFAULT_PROPS} title={'mock title'} />);
+      const title = container.getElementsByClassName(styles.Title);
+      expect(title.length).toBe(1);
+      expect(title[0]).toHaveTextContent('mock title');
+    });
+
+    test('should only apply Value class to only the value element', () => {
+      const { container } = render(<Card {...DEFAULT_PROPS} value={'mock value'} />);
+      const value = container.getElementsByClassName(styles.Value);
+      expect(value.length).toBe(1);
+      expect(value[0]).toHaveTextContent('mock value');
+    });
+
+    test('should only apply BottomText class to the bottom text element', () => {
+      const { container } = render(<Card {...DEFAULT_PROPS} bottomText={'mock bottom text'} />);
+      const bottomText = container.getElementsByClassName(styles.BottomText);
+      expect(bottomText.length).toBe(1);
+      expect(bottomText[0]).toHaveTextContent('mock bottom text');
+    });
+  });
+
+  describe('when text props are synchronous', () => {
+    test('should render title', () => {
+      render(<Card {...DEFAULT_PROPS} title={'mock title'} />);
+      expect(screen.getByText('mock title')).toBeInTheDocument();
```

Comment:
screen or 
```
const { getByText } = render(<Card {...DEFAULT_PROPS} bottomText={'mock bottom text'} />);
```

Not fussed myself, but we should move forward with a consistent rule

Reply:
We should [use screen](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen).

> The benefit of using screen is you no longer need to keep the render call destructure up-to-date as you add/remove the queries you need. You only need to type screen. and let your editor's magic autocomplete take care of the rest.
> The only exception to this is if you're setting the container or baseElement which you probably should avoid doing (I honestly can't think of a legitimate use case for those options anymore and they only exist for historical reasons at this point).
> You can also call [screen.debug](https://testing-library.com/docs/dom-testing-library/api-queries#screendebug) instead of debug

Reply:
Note: I know this one says not to destructure `{ container }` but I felt it was necessary on some of the tests. I would prefer not to add a test ID to the container if we can get the container directly from the `render` function.

Reply:
Sweet, sounds good to me!!